android-components/build.gradle

```diff
@@ -238,7 +238,7 @@ subprojects {
 
                 tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
                     kotlinOptions {
-                        jvmTarget = "1.8"
+                        jvmTarget = "11"
```

Comment:
@mcarare I think there are a few more occurrences of `jvmTarget` to update: https://github.com/mozilla-mobile/firefox-android/search?q=jvmTarget

The nimbus-gradle-plugin we will remove from this repo this week.

Reply:
(But looks like Focus files need an update.)

Reply:
I initially did not want to touch the `jvmTarget` for glean and telemetry and I only updated the tasks lint was complaining about. I will update the target for telemetry and glean also in a separate commit (so it can be easily reverted if any side-effects occur.)